stock_custom_exchange_rate_date/models/stock_picking.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    exchange_rate_date = fields.Date(
+        states={'done': [('readonly', True)], 'cancel': [('readonly', True)]},
```

Comment:
@luisg123v 

The fields should not be copy when there is a backorder, maybe only adding copy=False it is enough to backorder

Reply:
Done, thanks! Yes, that should be enough, as backorder is generated copying the original picking. That field should be copy=False anyway.